macs/README.md

```diff
@@ -0,0 +1,4 @@
+# Deploying to darwin
+
+Currently all mac-mini's are housed in [Grahams](https://github.com/grahamc) house.
+[@cole-h](https://github.com/cole-h) can deploy to these machines.
```

Comment:
Specifically, I have access to these macs:

* cosmic-stud
* tight-bug
* quality-ram
* becoming-hyena

Reply:
Added.